api/v1beta2/conversion_test.go

```diff
@@ -0,0 +1,52 @@
+package v1beta2_test
+
+import (
+	"testing"
+
+	mocov1beta1 "github.com/cybozu-go/moco/api/v1beta1"
+	mocov1beta2 "github.com/cybozu-go/moco/api/v1beta2"
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/go-cmp/cmp/cmpopts"
+	"k8s.io/apimachinery/pkg/api/apitesting/roundtrip"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+func TestCompatibility(t *testing.T) {
+	scheme := runtime.NewScheme()
+	_ = mocov1beta1.AddToScheme(scheme)
+	_ = mocov1beta2.AddToScheme(scheme)
+
+	f := roundtrip.CompatibilityTestFuzzer(scheme, nil)
+	f.NilChance(0.5).NumElements(0, 3)
+
+	t.Run("v1beta2 => v1beta1 => v1beta2", func(t *testing.T) {
```

Comment:
Using fuzzing to conversion APIs to each other and test that no information is missing.

Reply:
refs: https://www.youtube.com/watch?v=HsYtMvvzDyI&t=1237s